src/test/unit/controls/Graph/GraphAxes-spec.js

```diff
@@ -685,4 +693,205 @@ describe("Graph - Axes", () => {
             });
         });
     });
+    describe("Axis label", () => {
+        beforeEach(() => {
+            graph.destroy();
+            axisDefault.y2 = {
+                show: true,
+                label: "Some Y2 Label",
+                lowerLimit: 0,
+                upperLimit: 20
+            };
+        });
+        afterEach(() => {
+            axisDefault.y2 = "";
+        });
+        it("Truncates if too long", (done) => {
+            const labelAxisObj = utils.deepClone(axisDefault);
+            labelAxisObj.y.label = largeLabel;
+            labelAxisObj.y2.label = largeLabel;
+            labelAxisObj.x.label = largeLabel;
+            graph = new Graph(getAxes(labelAxisObj));
+            expect(
+                fetchElementByClass(styles.axisLabelX).querySelector("text")
+                    .textContent
+            ).toBe(
+                "Project long display value which is only for testing, Project long display value which is only for testing Project lon..."
+            );
+            expect(
+                fetchElementByClass(styles.axisLabelY).querySelector("text")
+                    .textContent
+            ).toBe("Project long display value w...");
+            expect(
+                fetchElementByClass(styles.axisLabelY2).querySelector("text")
+                    .textContent
+            ).toBe("Project long display value w...");
+            done();
+        });
```

Comment:
Can we write the test cases like this for different widths 10, 25, 50, 100, 150, 200, 250 and 300?
```js
Describe("if axes width is 50", () -> {
          it("should not truncate the x-axis label", (done) =>{
          }

          it("should truncate the y-axis label", (done) =>{
          }

          it("should not truncate the y-2-axis label", (done) =>{
          }
}

Describe("if axes width is 100", () -> {
          it("should not truncate the x-axis label", (done) =>{
          }

          it("should truncate the y-axis label", (done) =>{
          }

          it("should not truncate the y-2-axis label", (done) =>{
          }
}
```

Reply:
[fixed](https://github.com/cerner/carbon-graphs/pull/242/files#diff-11aa0aaf6db553889f349765e88f7462R749)